src/dev/src/app/slider/slider.demo.scss

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+h4 {
```

Comment:
all of that css should probably be removed

Reply:
why so?

Reply:
we talked about no longer using display: grid.

not quite sure right now regarding the h4 & horiziontal-center things, can take a look at this together

Reply:
ah right! i have overlooked those